db/changelog/v-1/10-changeset-project-table-insert.xml

```diff
@@ -14,6 +14,14 @@
             <column name="url" value="huaweiUrl" />
             <column name="description" value="test description" />
         </insert>
+        <insert dbms="mysql" tableName="project">
+            <column name="id" value="2" />
+            <column name="owner" value="Huawei" />
+            <column name="name" value="huaweiName 2" />
+            <column name="type" value="manual" />
+            <column name="url" value="huaweiUrl 2" />
```

Comment:
Let's use test data that looks like actual. You can always use `example.com` instead of actual urls, but randomly formatted string instead of url can cause some trouble if we use it in tests in the future

Reply:
Fixed